source/_posts/Synology-Bug-Bounty-2016.md

```diff
@@ -1,23 +1,41 @@
+---
 title: '[Synology Bug Bounty 2016]'
-author: 'bananaapple,0Alien0'
-date: 2017-03-20 07:28:13
+author: BambooFox
 tags:
+  - web
+  - 'Synology Bug Bounty '
+  - bug bounty
+  - DoS
+  - LFI
+  - command injection
+categories:
+  - write-ups
+  - bug-bounty-report
+date: 2017-03-20 07:28:00
 ---
-Synology Bug Bounty 
+Synology Bug Bounty Report
 =============
-Author: BambooFox
 
-In last year(2016), our team BambooFox is invited to Synology Bug Bounty. During about 2 mouth hacking, several vulnerabilities are discovered, include a remote root vuln. Enginners in Synology response and fix these vuln in a very sort time, which shows they pay a lot of attension for security issue.
+> Author: BambooFox Team 
+> ( Henry, jpeanut, ding, leepupu, Angelboy, boik, adr, Mango King, Bletchley )
+
+Last year ( 2016 ) , we BambooFox are invited to join the Synology Bug Bounty program. After about 2 months of hacking, we discovered several vulnerabilities, including a **remote root code execution** vulnerability. Synology engineers response and fix the vulnerabilities in a very sort time, which shows they pay a lot of attention to  security issues.
```

Comment:
I think it should be "we BambooFox were" since we use last year ?
And there is a redundant space in "to security issues".